.github/workflows/integrationTest.yml

```diff
@@ -675,6 +680,68 @@ jobs:
       - name: Terraform destroy
         run: terraform destroy --auto-approve
 
+  ECSEC2LaunchDaemonIntegrationTest:
```

Comment:
Not a problem to solve here, and not even sure if it's technically feasible, but it would be nice if we could separate out these integ test jobs because this YAML file is really long and hard to follow now.

Reply:
Yes. We can separate the workflow and use dispatch to trigger appropriate workflow https://github.com/peter-evans/repository-dispatch/releases

Reply:
We are not pulling in a dependency for an unverified GitHub action.

Reply:
Maybe I'm wrong but I think its has MIT License and also its only serve for the purpose of Github Action, not any Security Concern.